x-pack/plugins/alerts/server/usage/alerts_telemetry.test.ts

```diff
@@ -7,13 +7,13 @@
 import { getTotalCountInUse } from './alerts_telemetry';
 
 describe('alerts telemetry', () => {
-  test('getTotalCountInUse should replace action types names with . to __', async () => {
+  test('getTotalCountInUse should replace first "." symbol to "__" in alert types names', async () => {
     const mockEsClient = jest.fn();
     mockEsClient.mockReturnValue({
       aggregations: {
         byAlertTypeId: {
           value: {
-            types: { '.index-threshold': 2 },
+            types: { '.index-threshold': 2, 'logs.alert.document.count': 1 },
```

Comment:
Same comment as above...should we add a type that ends in a `.` to test that it gets stripped?